pkgs/fc/ceph/src/fc/ceph/main.py

```diff
@@ -440,8 +440,15 @@ def luks(args=sys.argv[1:]):
     parser_rekey = keystore_sub.add_parser("rekey", help="Rekey volumes.")
     parser_rekey.add_argument(
         "--lvs",
-        help="Names of encrypted LVs to update (globbing allowed), e.g. 'osd-1*'.",
-        default="*",
+        help="Names of encrypted LVs to update (globbing allowed), e.g. 'osd-1*'. Mutually exclusive with `--device`.",
+    )
+    parser_rekey.add_argument(
```

Comment:
For daily operations, having to *explicitly know per host* which individual lvs and/or blockdevices, and whether they explicitly require an external header, is not feasible. The variability is there in our deployments, but needs to be handled by the tooling, not the operator.

Idea:
- [ ] use `lsblk` output and the parents of `crypt` types
- [ ] for external headers, try re-opening the device with an external header of the file name scheme: `${mountpoint}.luks`, e.g. /srv/backy -> /srv/backy.luks
- [ ] when setting up a new backy volume with an internal header, clean up remaining external header backups?
- [ ] filtering: name of the crypted volume in lsblk
  - [ ] backy migration: ensure that the name of the migrated luks volume is something sensible, if not we need to add the luks labeling step to the migration